app/src/sharedTest/java/org/oppia/android/app/devoptions/MarkChaptersCompletedActivityTest.kt

```diff
@@ -750,6 +750,116 @@ class MarkChaptersCompletedActivityTest {
     }
   }
 
+  @Test
+  fun testMarkChaptersCompletedActivity_clickMarkCompleted_activityFinishes() {
+    launch<MarkChaptersCompletedActivity>(
+      createMarkChaptersCompletedActivityIntent(internalProfileId)
+    ).use {
+      testCoroutineDispatchers.runCurrent()
+      onView(withId(R.id.mark_chapters_completed_mark_completed_text_view)).perform(click())
+      assertThat(activityTestRule.activity).isNull()
```

Comment:
it should be something like 

example
```
assertThat(activityTestRule.activity.isFinishing).isTrue()
```

Reply:
Thanks for this. I have updated the tests. PTAL.